csharp/private/actions/assembly.bzl

```diff
@@ -87,7 +103,8 @@ def AssemblyAction(
     args.add_all(resources, map_each = _format_resource_arg)
 
     # defines
-    args.add_joined("/define", defines, join_with=";")
+    _add_framework_preprocessor_symbols(args, target_framework)
+    args.add_joined("/define", defines, join_with=":")
```

Comment:
Fixed a related typo here

Reply:
You sure?

Reply:
Yeah `;` vs `:`

Reply:
Ohhhh

Reply:
This is the separator for the different symbols

Reply:
Ok I think the old thing would do `/define A;B` but what we want is either `/define:A;B` or `/define:A /define:B`. I'll back this out and we can fix it later.

Reply:
I did the sad thing :(

Reply:
Looking at the code before, it definitely wasn't obvious that it supported `/define` being specified multiple times